pkg/mcs/resourcemanager/server/config.go

```diff
@@ -39,8 +39,14 @@ const (
 	defaultBackendEndpoints = "http://127.0.0.1:2379"
 	defaultListenAddr       = "http://127.0.0.1:3379"
 
-	defaultReadBaseCost  = 0.25
+	// 1 RU = 8 storage read requests
+	defaultReadBaseCost = 1. / 8
```

Comment:
Can we reuse definitions in server implementation instead of duplicate code here. 
It's very easy to forget updating whenever any of it is changing.

Reply:
Agree, I added TODO in line 114, and will do it in a separate pr.